src/partials/50_section-products.html

```diff
@@ -46,7 +46,10 @@ <h5>Aenean Ru Bristique</h5>
               <a href="#" class="btn-outline"><i class="far fa-heart"></i></a>
               <a href="#" class="btn-outline"><i class="fas fa-exchange-alt"></i></a>
             </div>
-            <div class="price"><div class="btn-main-small no-hover">$ 30.00</div></div>
+            <div class="price">
+              <div class="old-price-1 btn-outline no-hover">$ 50.00</div>
```

Comment:
Nie moze to byc tak zrobione - jako old-price-1,2,3 itd
Po pierwsze jedna klasa - old price - zeby wszystkie mialy style, no i tam gdzie ma sie wyrozniac, druga klasa ktora je odrozni

Reply:
sam jak na to patrzę teraz to się dziwię :) lepiej będzie zrobić drugą klasę, np. old-price-visible i oznaczać te, co trzeba. potem łatwiej to będzie obsłużyć JSem :) poprawiam!